test/unit/views/disassociateucr/DisassociateUcrViewSpec.scala

```diff
@@ -16,26 +16,44 @@
 
 package views.disassociateucr
 
-import base.Injector
-import forms.UcrType.{Ducr, Mucr}
+import base.{Injector, OverridableInjector}
+import config.DucrPartConfig
 import forms.DisassociateUcr
+import forms.UcrType.{Ducr, Mucr}
 import models.cache.DisassociateUcrAnswers
 import org.jsoup.nodes.Document
+import org.mockito.Mockito.{reset, when}
+import org.scalatest.BeforeAndAfterEach
+import org.scalatestplus.mockito.MockitoSugar
 import play.api.data.Form
+import play.api.inject.bind
 import play.twirl.api.Html
 import views.ViewSpec
 import views.html.disassociateucr.disassociate_ucr
 import views.tags.ViewTest
 
 @ViewTest
-class DisassociateUcrViewSpec extends ViewSpec with Injector {
+class DisassociateUcrViewSpec extends ViewSpec with MockitoSugar with BeforeAndAfterEach {
+
+  private val appConfig = mock[DucrPartConfig]
+  private val injector = new OverridableInjector(bind[DucrPartConfig].toInstance(appConfig))
 
   private implicit val request = journeyRequest(DisassociateUcrAnswers())
 
-  private val disassociatePage: disassociate_ucr = instanceOf[disassociate_ucr]
+  private val disassociatePage: disassociate_ucr = injector.instanceOf[disassociate_ucr]
 
   private def createView(form: Form[DisassociateUcr]): Html = disassociatePage(form)(request, messages)
 
+  override def beforeEach(): Unit = {
+    super.beforeEach()
+    when(appConfig.isDucrPartsEnabled).thenReturn(true)
+  }
```

Comment:
You can remove `beforeEach` You're defining this behaviour in test cases.

In general it's not a good to define some behaviour two times, could cause some weird issues.